packages/cos/os-release

```diff
@@ -0,0 +1,6 @@
+NAME="cOS"
+ID="cOS"
+ANSI_COLOR="0;32"
+BUG_REPORT_URL="https://github.com/mudler/cOS/issues"
```

Comment:
We will change this, probably by auto-generating it from the package data. just using it as placeholder mostly